Utilities/PlayerUtils.cs

```diff
@@ -322,6 +322,16 @@ public static bool CheckSolidGround(this Player player, int solidGroundAhead = 0
             }
             return ConditionMet;
         }
+
+        /// <summary>
+        /// Disables the default wing flap sound from vanilla; this is specifically to address issues with (currently) <see cref="CalamityMod.Items.Accessories.Wings.AureateBooster"/> and <see cref="CalamityMod.Items.Accessories.Wings.MOAB"/>.
+        /// </summary>
+        /// <param name="player">The Player to disable the sound on</param>
+        public static void DisableDefaultWingFlapSound(this Player player)
+        {
+            player.flapSound = true;
```

Comment:
Wouldn't hurt to leave a comment explaining that the sound only plays when this value is false (despite the name implying otherwise) instead of only mentioning it in the pr.